compiler/passes/convert-uast.cpp

```diff
@@ -642,7 +642,13 @@ struct Converter {
     astlocMarker markAstLoc(node->id());
 
     Expr* one = toExpr(convertAST(node->symbol()));
-    Expr* two = toExpr(convertAST(node->rename()));
+    Expr* two = nullptr;
+    // if this rename is to '_', don't convert it to chpl__tuple_blank!
+    if (node->rename()->toIdentifier()->name()==USTR("_"))
```

Comment:
This assumes that the rename is always going to be an identifier. I think that's a fair assumption, but in that case maybe we should add an assert, and just use the ident name directly rather than going through a convert call. 

```cpp
auto renameIdent = node->rename()->toIdentifier();
INT_ASSERT(renameIdent);
Expr* two = new UnresolvedSymExpr(renameIdent->name().c_str());
```

In the future I think we should streamline `use` and `import` in the converter, but we don't need to worry about that today.